dist/systemd/barrierc@.service.in

```diff
@@ -0,0 +1,59 @@
+#  Barrier Client service barrierc-@BARRIER_VERSION@
+#
+#  This file is part of Barrier.
+#
+#  This is a systemd template service for the barrierc client service. Instances
+#  are started using @<hostname>. For example:
+#
+#  Starting:
+#   systemctl start barrierc@myserver
+#  or:
+#   systemctl start barrierc@myserver:24800
+#
+#  Enabling:
+#   systemctl enable barrierc@myserver
+#
+#  SSL Fingerprints are stored in /var/lib/barrier@<hostname> for each client 
+#  instance.
+
+[Unit]
+Description=Barrier Client connected to %I (Open-source KVM software)
+Documentation=man:barrierc(1) man:barriers(1)
+Documentation=https://github.com/debauchee/barrier/wiki
+# Require network before starting barrierc
+After=network-online.target
+Wants=network-online.target
+# Don't run client if server is running
+Conflicts=barriers.service
+
+[Service]
+Type=exec
+# Log level may be FATAL, ERROR, WARNING, NOTE, INFO, DEBUG, DEBUG1, DEBUG2
+Environment=LOG_LEVEL=INFO
+# Default display is :0
+Environment=DISPLAY=:0
+# Store fingerprints in instance specific directories
+Environment=XDG_DATA_HOME=/var/lib/barrier/barrier@%i
+# TrustedServers.txt Directory
+Environment=FP_DIR=/var/lib/barrier/barrier@%i/barrier/SSL/Fingerprints
+# Ensure the Fingerprints directory exists
+ExecStartPre=mkdir -p "${FP_DIR}"
```

Comment:
Note that Systemd `Exec*` commands need to be an absolute path on the current LTS release of Ubuntu (18.04.3) which runs Systemd 237. From that version of the man page ("COMMAND LINES" section):

> The command to execute must be an absolute path name. It may contain spaces, but control characters are not allowed.

The current version of Systemd [seems to relax this](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Command%20lines):

> If the command is not a full (absolute) path, it will be resolved to a full path using a fixed search path determinted at compilation time. Searched directories include /usr/local/bin/, /usr/bin/, /bin/ on systems using split /usr/bin/ and /bin/ directories, and their sbin/ counterparts on systems using split bin/ and sbin/. It is thus safe to use just the executable name in case of executables located in any of the "standard" directories, and an absolute path must be used in other cases. Using an absolute path is recommended to avoid ambiguity. Hint: this search path may be queried using systemd-path search-binaries-default.

Reply:
Good point, I'll change it to absolute paths